google-cloud-bigtable/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Release History
 
+### 0.8.0 / 2019-11-01
+
+#### ⚠ BREAKING CHANGES
+
+* The following methods now raise Google::Cloud::Error instead of
+  Google::Gax::GaxError and/or GRPC::BadStatus:
+  * Table#mutate_row
+  * Table#read_modify_write_row
+  * Table#check_and_mutate_row
+  * Table#sample_row_keys
+
+#### Features
+
+* Raise Google::Cloud::Error from the following methods:
+  * Table#mutate_row
+  * Table#read_modify_write_row
+  * Table#check_and_mutate_row
+  * Table#sample_row_keys
```

Comment:
I don't think we need to repeat this. It's already present under the breaking changes section.

Reply:
Reverted.

Reply:
@dazuma did you want the features section removed, or the sub-list reverted?